src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonTableViewCell.swift

```diff
@@ -71,6 +71,7 @@ class HealthCertifiedPersonTableViewCell: UITableViewCell, ReuseIdentifierProvid
 		}
 
 		setupAccessibility(validityStateTitleIsVisible: cellModel.caption != nil)
+		captionCountLabel.heightAnchor.constraint(equalToConstant: captionCountLabel.intrinsicContentSize.height).isActive = true
```

Comment:
@ArturFriesen
We have a **horizontal** force that is **squeezing** the label, so the content hugging settings won't help here.
Also horizontal content compression is already setup as '.required' (we have an ENALabel here).
What we actually want is to prevent the very slim digit '1' from being squeezed beyond a **quadratic** size.

I try to make this intent clearer.
Please have a look at my latest commit.

Reply:
@ndegendogo the label is a subview of `captionCountView`. Did you try to set the compression resistance also on `captionCountView`?

Reply:
@ArturFriesen thanks for the suggestions - I'll try and report back

Reply:
@ArturFriesen wow - the following finally did the trick:
- a constraint to prevent **vertical** content compression on the captionCountLabel
- plus reducing the horizontal content compression prio for the captionLabel (.defaultHigh is the default value for iOS buttons)

Please have a look at my latest commit.